lib/fortnox/api/types/fortnox_country_code.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
This file has the wrong name. Also maybe put this in `lib/fortnox/api/types/shims/country_code_string.rb` just to underline that it is a special type of type? 🙂

Reply:
Correct, good catch! Should we keep this class or should we override the `class` method as we talked about?

Reply:
I think we can keep the class, it is cleaner - even if the other one is more fun 🙂